gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,8 +66,23 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
+      queue_failover_executed_event
+    end
+
     private
 
+    def queue_failover_executed_event
+      connection = pg_connection(@database_yml.pg_params_from_database_yml)
+      connection.exec(<<-SQL)
```

Comment:
Makes me nervous too. Not sure what other options there are if we do this in failover monitor. Is there are way that a running MiqServer can detect that a failover was done so that it can queue up the event the normal way through MiqQueue.put? I think @carbonin may have suggested that.